### PR TITLE
TIP-860: Move enrich component classes to the right BC

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-3.0.md [hunk 1]
```diff
@@ -1025,3 +1025,12 @@
 - Move `Pim\Bundle\EnrichBundle\EventListener\TranslateFlashMessagesSubscriber` to `Akeneo\Platform\Bundle\UIBundle\EventListener\TranslateFlashMessagesSubscriber`
 - Move `Pim\Bundle\EnrichBundle\EventListener\UserContextListener` to `Akeneo\Platform\Bundle\UIBundle\EventListener\UserContextListener`
 - Remove `Pim\Bundle\EnrichBundle\EventListener\RequestListener`
+- Remove `Pim\Component\Enrich\Model\ChosableInterface`
+- Move `Pim\Component\Enrich\Converter\ConverterInterface` to `Akeneo\Pim\Enrichment\Component\Product\Converter\ConverterInterface`
+- Move `Pim\Component\Enrich\Converter\EnrichToStandard\ValueConverter` to `Akeneo\Pim\Enrichment\Component\Product\Converter\EnrichToStandard\ValueConverter`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: InternalApiToStandard?

File: CHANGELOG-3.0.md [hunk 2]
```diff
@@ -1025,3 +1025,12 @@
...
+- Move `Pim\Component\Enrich\Converter\StandardToEnrich\ValueConverter` to `Akeneo\Pim\Enrichment\Component\Product\Converter\StandardToEnrich\ValueConverter`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: StandardToInternalApi?

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Job/DeleteProductsAndProductModelsTaskletSpec.php [hunk 3]
```diff
@@ -377,7 +377,7 @@ function it_throws_an_exception_if_step_execution_is_not_set()
                 new \InvalidArgumentException(
                     sprintf(
                         'In order to execute "%s" you need to set a step execution.',
-                        DeleteProductsAndProductModelsTasklet::class
+                        \Akeneo\Pim\Enrichment\Component\Product\Job\DeleteProductsAndProductModelsTasklet::class
```
Comment (jjanvier) [inline src/Akeneo/Pim/Enrichment/Component/spec/Product/Job/DeleteProductsAndProductModelsTaskletSpec.php]: no needed

File: CHANGELOG-3.0.md (hunk 1 above)
Comment (anaelChardan) [reply CHANGELOG-3.0.md]: I had the same question

**Pull request merged**